lib/presets/custom/hugo/deliver/prebuild.js

```diff
@@ -0,0 +1,21 @@
+import { readFile, rm } from 'fs/promises';
+import { exec, getPackageManager, copyDirectory } from '#utils';
+
+const packageManager = await getPackageManager();
+
+/**
+ * Runs custom prebuild actions
+ */
+async function prebuild() {
+  const newOutDir = '.edge/storage';
+  let outDir = 'public';
+
+  const command = packageManager === 'npm' ? 'npx' : packageManager;
+  await exec(`${command} hugo`, 'Hugo', true);
+
+  // move files to vulcan default path
+  copyDirectory(outDir, newOutDir);
+  rm(outDir, { recursive: true, force: true });
```

Comment:
We need to generate the manifest at the end of the prebuild step. You can copy this from https://github.com/aziontech/vulcan/blob/stage/lib/presets/custom/react/deliver/prebuild.js#L14 (lines 14-20) as this preset is purely static (deliver mode).